### PR TITLE
[doc](multi-catalog) paimon catalog doc

### PR DESCRIPTION
**Pull request by yuxuan-luo**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/docs/lakehouse/multi-catalog/paimon.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
```
Comment (morningman) [inline docs/en/docs/lakehouse/multi-catalog/paimon.md:1]: need add this new doc to `docs/sidebars.json`

Comment (yuxuan-luo) [reply docs/en/docs/lakehouse/multi-catalog/paimon.md:1]: > 

done

**Review by morningman — APPROVED** [review]
LGTM

**Comment by morningman** [comment]
run buildall

**Pull request merged**